smartapps/curb/curb-energy-monitor.src/curb-energy-monitor.groovy

```diff
@@ -0,0 +1,477 @@
+/**
+ *
+ *  Copyright 2017 Curb
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+include 'asynchttp_v1'
+
+definition(
+    name: "CURB Energy Monitor",
+    namespace: "curb",
+    author: "Curb",
+    description: "Gain insight into energy usage throughout your home.",
+    category: "Green Living",
+    iconUrl: "http://energycurb.com/wp-content/uploads/2015/12/curb-web-logo.png",
+    iconX2Url: "http://energycurb.com/wp-content/uploads/2015/12/curb-web-logo.png",
+    iconX3Url: "http://energycurb.com/wp-content/uploads/2015/12/curb-web-logo.png",
+    singleInstance: true
+) {
+    appSetting "clientId"
+    appSetting "clientSecret"
+}
+
+preferences {
+    page(
+        name: "auth",
+        title: "Curb",
+        nextPage: "",
```

Comment:
Since there is only one page, `nextPage: ""` is not necessary. Please remove this.